docs/core/compatibility/sdk/7.0/automatic-rid-publish-only.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Breaking change: Automatic RuntimeIdentifier for publish only"
+description: Learn about a breaking change in the .NET 7 SDK where a RuntimeIdentifier is automatically added to projects only for the dotnet publish command.
+ms.date: 01/12/2023
+---
+# Automatic RuntimeIdentifier for publish only
+
+In the 7.0.100 SDK, there was [a change](automatic-runtimeidentifier.md) to automatically add `<RuntimeIdentifier>` to projects with properties that require a runtime identifier (RID). Those properties are as follows:
+
+- `SelfContained`
+- `PublishAot`
+- `PublishReadyToRun`
+- `PublishSingleFile`
+- `PublishSelfContained`
+
+However, all of these properties except for `SelfContained` are only used for publishing. Yet the implicit `<RuntimeIdentifier>` was added for any [`dotnet` operation](../../../tools/dotnet.md) if these properties were in the project file or specified as part of the `dotnet` command.
+
+Now, the automatic RID for these properties is only added during publish. In addition, the automatic RID is only added when using the `dotnet publish` CLI command. It's not added when you publish from Visual Studio or `msbuild`, as those are separate mechanisms, and Visual Studio should provide its own RID.
```

Comment:
`Now, the automatic RID for these properties is only added during publish.` 


```suggestion
Now, the automatic RID for these properties, excluding `SelfConained`, is only added during publish. In addition, the automatic RID is only added when using the `dotnet publish` CLI command. It's not added when you publish from Visual Studio or `msbuild`, as those are separate mechanisms, and Visual Studio should provide its own RID.
```

It may be better to be explicit here.